src/client/src/tests/components/LoginForm.test.tsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+import { render, fireEvent, act } from '@testing-library/react';
+import { LoginForm } from '../../components/LoginForm';
+import { BrowserRouter } from 'react-router-dom';
+
+const getComponent = (loginUser?: any) => (
+    <BrowserRouter>
+        <LoginForm loginUser={loginUser || jest.fn()} setUser={jest.fn()} />
+    </BrowserRouter>
+);
+
+describe('<LoginForm>', () => {
+    test('is visible to users', async () => {
+        let loginForm: any;
+        act(() => {
+            loginForm = render(getComponent());
+        });
+
+        /* expect(loginForm.container).toHaveTextContent('Email');
+        expect(loginForm.container).toHaveTextContent('Password'); */
+
+        const input = loginForm.container.querySelectorAll('input');
+        expect(input).toHaveLength(2);
+    });
+
+    test('writing into the inputs should be possible', async () => {
+        let loginForm: any;
+        act(() => {
+            loginForm = render(getComponent());
+        });
+
+        const input = loginForm.container.querySelectorAll('input');
+
+        act(() => {
+            fireEvent.change(input[0], {
+                target: { value: 'mrtest@nodes.com' },
```

Comment:
All example emails should be `@example.com`, so that we won't accidentally send email to a 'real' address (in this case, that looks like some blockchain company)